### PR TITLE
[ADD] hr_cae_contract: ongoing contract types tags on employees

### PR DESCRIPTION
**Pull request by mclaeysb**
Adds colored 'ongoing contract types tags' on employees, at the same place of category_ids.

Only disadvantage could be that with category_id and this next to each other there could be confusion.

### CODE REVIEW

**Review by robinkeunen — APPROVED** [review]
LGTM 👍

**Comment by quentinDupont** [comment]
> Adds colored 'ongoing contract types tags' on employees, at the same place of category_ids.
> 
> Only disadvantage could be that with category_id and this next to each other there could be confusion.

I add two contracts for this employee, one with main echelon and the other is an amendment.
But i'm suppose to have new fields here right ?
image

**Comment by mclaeysb** [comment]
The field is right next to (not below) the categories. The tags should appear once the contracts are in an 'ongoing' state.

File: hr_cae_contract/models/hr_employee.py
Comment (quentinDupont) [inline hr_cae_contract/models/hr_employee.py:30]: Should be `and c.echelon in ["main", "amendment", "termination"]` if we want termination contract to be displayed

**Review by quentinDupont — CHANGES_REQUESTED** [review]

File: hr_cae_contract/models/hr_employee.py
Comment (mclaeysb) [reply hr_cae_contract/models/hr_employee.py:30]: Indeed, but this was not requested by our client, and I don't think it makes sense. Termination contracts serve to terminate main and amendment contracts, these tags serve to show which main or amendment are ongoing.

Comment (quentinDupont) [reply hr_cae_contract/models/hr_employee.py:30]: Okay indeed I understand
It was confusing for me because of [that line](https://github.com/odoo-cae/odoo-addons-hr-incubator/blob/230d4229139c07002bcad0969f9d3c0cd7d85ecb/hr_cae_contract/models/hr_contract.py#L87) that doesn't seem useful.
Do you want to keep it ?

Comment (mclaeysb) [reply hr_cae_contract/models/hr_employee.py:30]: I just wanted to make a generic future proof function :)

**Review by quentinDupont — APPROVED** [review]

**Comment by robinkeunen** [comment]
@mclaeysb ready to merge?

**Comment by mclaeysb** [comment]
Seems like it, I'm verifying...

**Comment by mclaeysb** [comment]
Still waiting for client validation before we can merge this.

**Pull request merged**
